examples/jsm/nodes/Nodes.js

```diff
@@ -111,6 +111,8 @@ export { default as ViewportSharedTextureNode, viewportSharedTexture } from './d
 export { default as ViewportDepthTextureNode, viewportDepthTexture } from './display/ViewportDepthTextureNode.js';
 export { default as ViewportDepthNode, viewZToOrthographicDepth, orthographicDepthToViewZ, viewZToPerspectiveDepth, perspectiveDepthToViewZ, depth, depthTexture, depthPixel } from './display/ViewportDepthNode.js';
 export { default as GaussianBlurNode, gaussianBlur } from './display/GaussianBlurNode.js';
+export { default as AfterImageNode, afterImage } from './display/AfterImageNode.js';
```

Comment:
@sunag Maybe we can create a new directory for postprocessing nodes (GaussianBlur, AfterImage), like `postprocessing/`?

Reply:
Interesting, `filters` seem to sound better to me, I don't think they should be for exclusive use in post-processing.